set.mm

```diff
@@ -177175,23 +177173,21 @@ obviously not a bijection (by Cantor's theorem ~ canth2 ), and in fact
        This is Metamath 100 proof #69 (greatest common divisor algorithm).
        (Contributed by Paul Chapman, 31-Mar-2011.)  (Proof shortened by Mario
        Carneiro, 29-May-2014.) $)
-    eucalg $p |- ( ( M e. NN0 /\ N e. NN0 ) ->
-                   ( 1st ` ( R ` N ) ) = ( M gcd N ) ) $=
-      ( cn0 wcel cfv cgcd cc0 c2nd wceq syl fveq2d cz vz wa co cop cxp wf nn0uz
-      c1st 0zd opelxpi syl5eqel eucalgf a1i algrf ffvelrn 1st2nd2 df-ov syl6eqr
-      sylancom fveq2i op2ndg syl5eq cuz xp2nd nn0zd uzid eqid eucalgcvga eqtr3d
-      mpd oveq2d xp1st nn0gcdid0 3syl 3eqtrrd cres wfn gcdf ax-mp nn0ssz xpss12
-      wss mp2an fnssres cv eucalginv ffvelrni fvres 3eqtr4d alginv 0nn0 sylancl
-      ffn 3eqtr3d algr0 syl6eq 3eqtrd ) FKLZGKLZUBZGDMZUHMZXANMZODMZNMZFGNUCZWT
-      XCXBXAPMZNUCZXBONUCZXBWTXCXBXGUDZNMXHWTXAXJNWTXAKKUEZLZXAXJQWRWSKXKDUFZXL
-      WTCDXKEOKUGIWTUIZWTCFGUDZXKJFGKKUJUKZXKXKEUFWTABEHULZUMUNZKXKGDUOUSZXAKKU
-      PRSXBXGNUQURWTXGOXBNWTCPMZDMZPMZXGOWTYAXAPWTXTGDWTXTXOPMGCXOPJUTFGKKVAVBS
-      SWTCXKLZYBOQZXPYCXTXTVCMLZYDYCXTTLYEYCXTCKKVDVEXTVFRABCDEXTXTHIXTVGVHVJRV
-      IVKWTXLXBKLXIXBQXSXAKKVLXBVMVNVOWTXANXKVPZMZXDYFMZXCXEWRWSYCYGYHQXPUACDXK
-      EYFGIXQNTTUEZVQZXKYIWBZYFXKVQYIKNUFYJVRYIKNWMVSKTWBZYLYKVTVTKTKTWAWCYIXKN
-      WDWCUAWEZXKLZYMEMZNMZYMNMYOYFMZYMYFMABEYMHWFYNYOXKLYQYPQXKXKYMEXQWGYOXKNW
-      HRYMXKNWHWIWJUSWTXLYGXCQXSXAXKNWHRWTXDXKLZYHXEQWTXMOKLYRXRWKKXKODUOWLXDXK
-      NWHRWNWTXEXONMXFWTXDXONWTXDCXOWTCDXKEOKUGIXNXPWOJWPSFGNUQURWQ $.
+    eucalg $p
+        |- ( ( M e. NN0 /\ N e. NN0 ) -> ( 1st ` ( R ` N ) ) = ( M gcd N ) ) $=
+  ( cn0 wcel cfv cgcd cc0 c2nd wceq syl fveq2d fvres vz wa c1st co cop wf nn0uz
+  cxp 0zd opelxpi syl5eqel eucalgf algrf ffvelrn sylancom 1st2nd2 df-ov syl6eqr
+  a1i fveq2i op2ndg syl5eq cuz cz xp2nd nn0zd uzid eucalgcvga mpd eqtr3d oveq2d
+  eqid xp1st nn0gcdid0 3syl 3eqtrrd cres eucalginv ffvelrni 3eqtr4d alginv 0nn0
+  cv sylancl 3eqtr3d algr0 syl6eq 3eqtrd ) FKLZGKLZUBZGDMZUCMZWLNMZODMZNMZFGNUD
+  ZWKWNWMWLPMZNUDZWMONUDZWMWKWNWMWRUEZNMWSWKWLXANWKWLKKUHZLZWLXAQWIWJKXBDUFZXCW
+  KCDXBEOKUGIWKUIZWKCFGUEZXBJFGKKUJUKZXBXBEUFWKABEHULZUSUMZKXBGDUNUOZWLKKUPRSWM
+  WRNUQURWKWROWMNWKCPMZDMZPMZWROWKXLWLPWKXKGDWKXKXFPMGCXFPJUTFGKKVAVBSSWKCXBLZX
+  MOQZXGXNXKXKVCMLZXOXNXKVDLXPXNXKCKKVEVFXKVGRABCDEXKXKHIXKVLVHVIRVJVKWKXCWMKLW
+  TWMQXJWLKKVMWMVNVOVPWKWLNXBVQZMZWOXQMZWNWPWIWJXNXRXSQXGUACDXBEXQGIXHUAWCZXBLZ
+  XTEMZNMZXTNMYBXQMZXTXQMABEXTHVRYAYBXBLYDYCQXBXBXTEXHVSYBXBNTRXTXBNTVTWAUOWKXC
+  XRWNQXJWLXBNTRWKWOXBLZXSWPQWKXDOKLYEXIWBKXBODUNWDWOXBNTRWEWKWPXFNMWQWKWOXFNWK
+  WOCXFWKCDXBEOKUGIXEXGWFJWGSFGNUQURWH $.
```

Comment:
I seems that there's an issue with formatting here: there is no indentation of the proof! I think this is because of having the ` |- ` in an extra line. Maybe  ` |- ` should/must follow the ` $p ` in the same line (as for ~alginv):

    eucalg $p |-
           ( ( M e. NN0 /\ N e. NN0 ) -> ( 1st ` ( R ` N ) ) = ( M gcd N ) ) $=

Reply:
You're right, this looks like a problem with set.mm formatting, and when I write it with |- on the first line, correct indentation resumes. I'm opening an Issue.